src/main/scala/ai/privado/languageEngine/java/passes/config/PropertiesFilePass.scala

```diff
@@ -87,6 +89,17 @@ class PropertiesFilePass(cpg: Cpg, projectRoot: String) extends ForkJoinParallel
           builder.addEdge(param, propertyNode, EdgeTypes.ORIGINAL_PROPERTY)
         }
     }
+    val membersAndValues = annotatedMembers()
```

Comment:
@rahul-privado also add the respective test cases for the member cpg along with small example in src/test/scala - ai.privado.languageEngine.java.passes.config.PropertiesFilePass

Reply:
Done